docs/sources/datasources/influxdb/configure-influxdb-data-source/_index.md

```diff
@@ -0,0 +1,269 @@
+---
+aliases:
+  - ../data-sources/influxdb/
+  - ../data-sources/influxdb/provision-influxdb/
+  - ../features/datasources/influxdb/
+  - provision-influxdb/
+description: Guide for using InfluxDB in Grafana
+keywords:
+  - grafana
+  - influxdb
+  - guide
+  - flux
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Configure the InfluxDB data source
```

Comment:
Maybe can just be "Configure" for a shorter item in the table of contents? The rest of the title is implied by the table of contents hierarchy

```suggestion
menuTitle: Configure
```

Reply:
I’d prefer to keep it as is. Users have provided feedback that the data source documentation is inconsistent and makes it difficult to find the information they need.